### PR TITLE
Add background removal plugin

### PR DESCRIPTION
**Pull request by mhcerri**
This pull request adds the obs-backgroundremoval plugin and also libonnxruntime, which is used as its dependency.

I've been using this plugin lately and it has worked very well. I created the new snapcraft parts using the instructions provided by the author in the git repo.

### CODE REVIEW

**Review by diddledani — DISMISSED** [review]

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -1111,6 +1111,44 @@ parts:
     prime:
       - -usr/lib/$SNAPCRAFT_ARCH_TRIPLET/obs-plugins
 
+  onnxruntime:
+    plugin: dump
+    source: https://github.com/microsoft/onnxruntime/releases/download/v1.7.0/onnxruntime-linux-x64-1.7.0.tgz
```
[the remark this answers is not in the document]
Comment (mhcerri) [reply snap/snapcraft.yaml]: Sorry for the delay. That was the version listed in the project, but I updated it to 1.9.0 and it works fine.

**Comment by mhcerri** [comment]
Hi, sorry for the delay. I updated the commit with the suggested changes and rebased it to the latest master. Let me know what you think.

**Comment by mhcerri** [comment]
@diddledani @flexiondotorg anything pending on that? I think all the changes are already addressed, correct? Please let me know if I need to change something else. Thanks!

**Review by flexiondotorg — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**